lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -61,7 +63,7 @@ def __init__(
             if axis_aspect_ratio
             else None
         )
-        self.lines = KymoLineGroup([])
+        self.lines_history = UndoStack(KymoLineGroup([]))
```

Comment:
I would make this an internal attribute to be on the safe side

Reply:
Good point! I'll amend it!